server_environment_ir_config_parameter/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Server Environment Ir Config Parameter
+======================================
+
+Override System Parameters from server environment file.
+Before using this module, you must be familiar with the
+server_environment module.
+
+Installation
+============
+
+There is no specific installation instruction for this module.
+
+Configuration
+=============
+
+To configure this module, you need to add a section ``[ir.config_parameter]`` to
+you server_environment_files configurations, where the keys are the same
+as would normally be set in the Systems Parameter Odoo menu.
+
+When first using a value, the system will read it from the configuration file
+and override any value that would be present in the database, so the configuration
+file has precedence.
+
+The user cannot write, create, or delete System Parameters that are defined in the
+configuration files.
```

Comment:
@moylop260 IMO one advantage of the conf file is that it's easily understood by system administrators and less errors prone than xml for this kind of audience. I like the idea of being able to display the environment ribbon just by specifying a value in my configuration file.